projects/kit/components/input-tag/input-tag.template.html

```diff
@@ -47,6 +47,7 @@
                 >
                     <div
                         class="t-tags"
+                        [class.t-has-native-placeholder]="hasNativePlaceholder"
```

Comment:
Can be host binding on placeholder input, `.t-with-placeholder` for brevity

Reply:
fixed